### PR TITLE
Update topnav styles and add npm scripts to compile css artifacts

### PR DESCRIPTION
**Pull request by RequiemOfSpirit**

### CODE REVIEW

File: http/templates/base.html.tera [hunk 1]
```diff
@@ -2,40 +2,36 @@
 {% block templates %}
 {% endblock %}
 <html lang="en">
-	<head>
```
Comment (RequiemOfSpirit) [inline http/templates/base.html.tera:5]: There seemed to be a mix of tab and space indenting here and the rest of the repo seemed to use spaces so I changed everything to spaces

**Comment by FoxLisk** [comment]
this looks good, thanks!

are you going to continue? i'd rather merge the styling changes all at once if there's more coming.

It might be nice to have the current tab highlighted visually?

**Comment by RequiemOfSpirit** [comment]
> this looks good, thanks!
> 
> are you going to continue? i'd rather merge the styling changes all at once if there's more coming.
> 
> It might be nice to have the current tab highlighted visually?

I pushed some changes for the season page to make the nac for the seasons specific pages look better. I may have time to make some quick changes on the home page and quals page, but after that i won't be able to work for another week.

I can work on the visual highlights for active pages when I get back as well

File: http/templates/season_base.html.tera [hunk 2]
```diff
@@ -1,29 +1,18 @@
...
-            <a href="/season/{{season.id}}/standings">Standings</a>
-        </div>
-        <div class="flex flex-auto">
-            <a href="/season/{{season.id}}/qualifiers">Qualifiers</a>
-        </div>
-        <div class="flex md:w-1/5">
-        </div>
+<div id="season-container">
+    <div id="season-banner">
+        <div id="season-title">Season {{base_context.current_season.id}} - {{ season.format }}</div>
```
Comment (FoxLisk) [inline http/templates/season_base.html.tera:6]: this is wrong for 2 reasons:

1. it uses `current_season` but it needs to use the id of the season we're looking at
2. database id is not the correct ordinal of the season anyway (i didn't burn an id, so database id 1 is the second season of league; i try to just avoid using the id in the UI at all)

also for dumb historical reasons `format` is really more like `title` - the "format" of season 2 is just "Season 2"

**Review by FoxLisk — COMMENTED** [review]
one big issue with the season title, otherwise looks good. Thanks!

**Pull request merged**